### PR TITLE
add Qonto as user

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Thomas Labarussias <thomas.labarussias@qonto.com>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

This PR adds Qonto, the company I'm working for, in list of users

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -356,6 +356,10 @@ logo = "pro-logo.svg"
 url = "https://www.purefacts.com"
 logo = "purefacts.png"
 
+[[params.qonto]]
```
Comment (tomkerkhove) [inline config.toml:359]: This will need to be aligned
```suggestion
[[params.users]]
```

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: config.toml (hunk 1 above)
Comment (Issif) [reply config.toml]: Sorry, too tired. I fixed and rebased. Thank you.

**Review by tomkerkhove — APPROVED** [review]
Looks perfect, thanks a ton! 🎉

**Comment by victorboissiere** [comment]
@tomkerkhove I'll answer for Qonto 👌

**Comment by tomkerkhove** [comment]
Thank you!

**Pull request merged**